containers/datalab/content/setup-env.sh

```diff
@@ -27,3 +27,6 @@ export CLOUDSDK_CONFIG=/content/datalab/.config
 export PROJECT_ID=${PROJECT_ID:-`gcloud config list -q --format 'value(core.project)' 2> /dev/null`}
 export ZONE=${ZONE:-`gcloud config list -q --format 'value(compute.zone)' 2> /dev/null`}
 
+# Lookup the author email address to use for git commits, and then configure git accordingly
+export GIT_AUTHOR=${DATALAB_GIT_AUTHOR:-`gcloud auth list --format 'value(account)' --filter 'status:ACTIVE'`}
```

Comment:
I don't agree that using the service account is a bad config.

It may be ugly, but it is technically correct. Additionally, advanced users may create descriptively named service accounts for which the service account is actually the best email address to use.

More importantly, we don't expect that case to actually happen.

Users will usually use the CLI to create Datalab instances running in GCE VMs, and that tool will set the right environment variable.

That means the case where this will happen is for someone running Datalab locally, and in that case their gcloud credentials should be the right email address to use.